examples/DateRangePickerWrapper.jsx

```diff
@@ -24,6 +24,8 @@ const propTypes = {
     'focusedInput',
     'onFocusChange',
   ]),
+
+  isRTL: PropTypes.bool, // sag1v rtl
```

Comment:
yes of course. these comments are my way of keeping track for the changes i made in case i need to undo or something,
as this is my first time working with github and i wasn't sure how the compare will work here.